lib/host/wasi_nn/wasinnfunc.cpp

```diff
@@ -6,31 +6,320 @@
 #include "runtime/hostfunc.h"
 #include "runtime/instance/memory.h"
 
+#ifdef WASMEDGE_WASINN_BUILD_OPENVINO
+#include <c_api/ie_c_api.h>
+#include <string>
+#endif
 namespace WasmEdge {
 namespace Host {
 
+const std::string map_target_to_string(uint32_t Target) {
+  switch (Target) {
+  case 0:
+    return "CPU";
+  case 1:
+    return "GPU";
+  case 2:
+    return "TPU";
+  default:
+    return "";
+  }
+}
+
 Expect<uint32_t> WasiNNLoad::body(Runtime::Instance::MemoryInstance *MemInst
                                   [[maybe_unused]],
                                   uint32_t BuilderPtr [[maybe_unused]],
                                   uint32_t BuilderLen [[maybe_unused]],
                                   uint32_t Encoding [[maybe_unused]],
                                   uint32_t Target [[maybe_unused]],
                                   uint32_t GraphPtr [[maybe_unused]]) {
-  return 0;
+  // GraphBuilders' Layout: |builder-0|builder-0 len|builder-1|builder-1 len|...
+  // Check memory instance from module.
+  if (MemInst == nullptr) {
+    return Unexpect(ErrCode::ExecutionFailed);
+  }
+  uint32_t *GraphBuilders;
+  uint32_t *Graph;
```

Comment:
Rename it. The `Graph` is defined as `uint32_t` in the header: `using Graph = uint32_t;`